storybook/preview.js

```diff
@@ -15,6 +15,82 @@ import cem from './custom-elements.json';
 
 setCustomElementsManifest(cem);
 
+export const globalTypes = {
+    system: {
+        title: 'Design context',
+        description: 'The variation of Spectrum to use in the component',
+        defaultValue: 'spectrum',
+        type: 'string',
+        showName: true,
+        toolbar: {
+            items: [
+                { value: 'spectrum', title: 'Spectrum 2', right: 'default' },
+                { value: 'legacy', title: 'Spectrum 1', right: 'legacy' },
+                { value: 'express', title: 'Express' },
+            ],
+            dynamicTitle: true,
+        },
+    },
+    color: {
+        title: 'Color',
+        description: 'Controls the color context of the component',
+        defaultValue: 'light',
+        icon: 'paintbrush',
```

Comment:
are these icons imported with chromatic?

Reply:
Those are built-in storybook icons made for use by the UI